chapters/categorical-predictors.qmd

```diff
@@ -0,0 +1,718 @@
+
+* Agent `r example_2$agent[1]` had `r format(example_2$n, big.mark = ",")` bookings with a simple mean ADR estimate of  `r round(example_2$mean_raw, 1)`€. We would judge the quality of this agent's data to be better than cases with a handful of bookings. 
+
+How well can we estimate an effect if there are only one or two bookings for an agent in the data set?
+
+Both of these concerns can be addressed by _how_ we estimate the effect of the predictor. Recall that @tbl-one-hot showed how simple linear regression was used to calculate the mean for each customer type. The way that the indicator variables impact the parameter estimates is such that the rows for each customer type are the only data used to estimate that customer type's effect. For example, the contract data do not directly affect the parameter estimate for the transient customers, and so on. This is often called a _no pooling_ method of estimating parameters. In this context, "pooling" means combining data across multiple categories to estimate parameters.  
+
+We could compute the naive ADR estimate using sample means and produce a similar table. We can create better mean estimates for each agent by considering the data from _all_ agents using a technique called _partial pooling_ [@gelman1995bayesian; @BayesRules,  chap. 15]. 
+
+::: {.mathy-box}
+In the following two sub-sections, we'll describe this technique in detail for cases where the outcome is a numeric value assumed to be normally distributed or or non-Gaussian. These sections will rely more on probability theory and statistical details than most others.
+:::
+
+#### Numeric outcomes {-}
+
+For our ADR analysis, the outcome is a quantitative value. A linear regression model (see @sec-linear-regression) is the most commonly used approach to model this type of outcome. Linear regression models the outcome as a function of a slope and intercept model. For our example, where we only consider estimating the effect of the agents using indicator columns, the model is:
+
+$$
+y_{ij} =\beta_0 + \beta_1x_{i1} + \ldots + \beta_{`r nrow(blups) -1`}x_{i `r nrow(blups) -1`} + \epsilon_{ij} = \mu_i  + \epsilon_{ij}
+$$
+
+where $y_{ij}$ is the outcome (in this case, the ADR of booking $i$ by agent $j$),  $x_{ij}$ is an indicator for reservation $i$ from agent $j$,  $\beta_0$ is the estimated outcome of the reference cell, and we assume $\epsilon_i \sim N(0, \sigma^2_\epsilon)$. This leads to the assumption that $y_{ij} \sim N(\mu_i, \sigma^2_\epsilon)$. 
+ The $\hat{\beta}_{1_j}$ parameters are the effect of the agents above and beyond the reference cell effect. The index $j$ is from 1 to `r nrow(blups) -1` since there were `r nrow(blups)` agents in the training set with at least one booking. 
```

Comment:
Should $\hat{\beta}_{1_j}$ be $\hat{\beta}_{j}$. ?

Reply:
Yep! Good eye!